<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filter</class>
 <widget class="QDialog" name="filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="orig_pic_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="orig_pic_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>             原始图</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="guass_widget" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="guass_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>              平滑去噪</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="sharp_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="sharp_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string>              锐化</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>530</y>
     <width>77</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Select_Pic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>选择图片</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="filter_comboBox">
      <item>
       <property name="text">
        <string>   高斯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>   中值</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>   双边</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>   均值</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sharp_comboBox">
      <item>
       <property name="text">
        <string>原图锐化</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>平滑锐化</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
